content/docs/command-reference/commit.md

```diff
@@ -1,7 +1,7 @@
 # commit
 
-Record changes to DVC-tracked files in the <abbr>project</abbr>, by saving them
-to the <abbr>cache</abbr> and updating the `dvc.lock` or `.dvc` files.
+<abbr>Cache</abbr> changed files and directories tracked by DVC, and update the
+corresponding `dvc.lock` or `.dvc` files if needed.
```

Comment:
> it was more explicit and human friendly

"Record changes to DVC-tracked files in the project" ? That's a kind of complicated way to say what commit does: store files in the cache.

Changed to "Stores changed changed files and directories tracked by DVC in the cache" for now.

Reply:
Moved to https://github.com/iterative/dvc.org/pull/1989/files#diff-b8f371cb91c101e733f37e95de023afe203b77b8a0443b3d453a76739b675852L3-R4